web/src/components/shared/modals/styles/HelmDeployModal.scss

```diff
@@ -0,0 +1,3 @@
+.helm pre {
```

Comment:
we'll use index files instead of `/style` folders when we refactor the file structure of the project to match BPR